src/System.Drawing.Common/tests/Graphics_DrawBezierTests.cs

```diff
@@ -0,0 +1,214 @@
+                {
+                    new Point(10, 10), new Point(20, 1), new Point(35, 5), new Point(50, 10),
+                    new Point(60, 15), new Point(65, 25), new Point(50, 30)
+                };
+
+                graphics.DrawBeziers(pen, points);
+                ValidateImageContent(image,
+                    PlatformDetection.IsWindows
+                        ? new byte[] { 0xd0, 0x00, 0x08, 0x21, 0x06, 0x29, 0xd8, 0xab, 0x19, 0xc5, 0xc9, 0xf6, 0xf2, 0x69, 0x30, 0x1f }
+                        : new byte[] { 0x9d, 0x24, 0x9f, 0x91, 0xa3, 0xa5, 0x60, 0xde, 0x14, 0x69, 0x42, 0xa8, 0xe6, 0xc6, 0xbf, 0xc9 });
+            }
+        }
+
+        private void ValidateImageContent(Image image, byte[] expectedHash)
+        {
+            using (MemoryStream stream = new MemoryStream(4096))
+            {
+                image.Save(stream, ImageFormat.Bmp);
+                stream.Seek(0, SeekOrigin.Begin);
+                byte[] hash = s_md5.ComputeHash(stream);
+                Assert.Equal(expectedHash, hash);
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawBezier_NullPen_ThrowsArgumentNullException()
+        {
+            using (var image = new Bitmap(10, 10))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                AssertExtensions.Throws<ArgumentNullException>("pen", () => graphics.DrawBezier(null, 1, 2, 3, 4, 5, 6, 7, 8));
+                AssertExtensions.Throws<ArgumentNullException>("pen", () => graphics.DrawBezier(null, Point.Empty, Point.Empty, Point.Empty, Point.Empty));
+                AssertExtensions.Throws<ArgumentNullException>("pen", () => graphics.DrawBezier(null, PointF.Empty, PointF.Empty, PointF.Empty, PointF.Empty));
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawBezier_DisposedPen_ThrowsArgumentException()
+        {
+            using (var image = new Bitmap(10, 10))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                var pen = new Pen(Color.Red);
+                pen.Dispose();
+
+                AssertExtensions.Throws<ArgumentException>(null, () => graphics.DrawBezier(pen, 1, 2, 3, 4, 5, 6, 7, 8));
+                AssertExtensions.Throws<ArgumentException>(null, () => graphics.DrawBezier(pen, Point.Empty, Point.Empty, Point.Empty, Point.Empty));
+                AssertExtensions.Throws<ArgumentException>(null, () => graphics.DrawBezier(pen, PointF.Empty, PointF.Empty, PointF.Empty, PointF.Empty));
+            }
+        }
+
+        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawBezier_Busy_ThrowsInvalidOperationException()
+        {
+            using (var image = new Bitmap(10, 10))
+            using (Graphics graphics = Graphics.FromImage(image))
+            using (var pen = new Pen(Color.Red))
+            {
```

Comment:
This hdc ought to have a disposable wrapper perhaps

Reply:
The finally releases it, but it might be nice to add a helper.